heartbeat/NodeUtilization

```diff
@@ -135,10 +135,9 @@ Host_Total_Memory() {
 
     xentool=$(which xl 2> /dev/null || which xm 2> /dev/null)
 
-    if [ -x $xentool ]; then
-        $xentool info | awk '/total_memory/{printf("%d\n",$3);exit(0)}'
+    if [ -x "$xentool" ]; then
+        "$xentool" info | awk '/total_memory/{printf("%d\n",$3);exit(0)}'
     else
-        ocf_log warn "Can only set hv_memory for Xen hypervisor"
```

Comment:
You should change this from `warn` to `debug`. That way it'll only show if you increase debugging or do e.g. `pcs resource debug-<action> --full <resource>`.

Reply:
Thanks, fixed in the updated patch.